src/olympia/translations/widgets.py

```diff
@@ -101,12 +104,16 @@ def delete_locale(s):
         return rv
 
     def format_output(self, widgets):
-        s = super(TransMulti, self).format_output(widgets)
+        # Gather output for all widgets as normal...
+        rendered_widgets = super(TransMulti, self).format_output(widgets)
+        # ...But also add a widget that'll be cloned for when we want to add
+        # a new translation. Hide it by default.
```

Comment:
How does the page allow new translations to be added?  It wasn't obvious from the screenshot (and nothing in your .js so I assume it's built-in functionality?)

Reply:
It's not possible in the admin - only possible through devhub (which uses a different bit of javascript) - I thought it wasn't as useful in the admin.

Reply:
I agree about it not being that useful - from reading the comment I wasn't sure if admin _did_ allow you to do it though.  Nit on comment, then.

Reply:
Tweaked the comment in https://github.com/mozilla/addons-server/pull/8465/commits/ca9a7c706c1ef1bc2331ef48f59f7879e8818b0e